client/me/concierge/shared/closure-notice.js

```diff
@@ -46,7 +46,7 @@ const ClosureNotice = ( { closesAt, displayAt, holidayName, reopensAt } ) => {
 		);
 	} else {
 		message = translate(
-			'{{strong}}Note:{{/strong}} Support sessions are closed for %(holidayName)s and will reopen %(reopensAt)s. ' +
+			'{{strong}}Note:{{/strong}} Quick Start sessions are closed for %(holidayName)s and will reopen %(reopensAt)s. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( '{{strong}}Note:{{/strong}} Support sessions are closed for %(holidayName)s and will reopen %(reopensAt)s. If you need to get in touch with us, you’ll be able to {{link}}submit a support request{{/link}} and we’ll get back to you as fast as we can. Thank you!' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/xmas-quick-start-closure-2019)